dev-lisp/clisp/clisp-2.49.95~git.recipe

```diff
@@ -0,0 +1,168 @@
+SUMMARY="ANSI Common Lisp Implementation"
+desc_base="GNU CLISP is an ANSI Common Lisp Implementation. \
+It is known for high portability, small image size and efficient bignums.
+
+This package contains "
+DESCRIPTION="${desc_base}base clisp executable and modules: \
+asdf, rawsock, clx/mit-clx, editor, queens."
+# TODO Add i18n regexp syscalls readline
+DESCRIPTION_berkeley_db="${desc_base}berkeley-db module."
+DESCRIPTION_gdbm="${desc_base}gdbm module."
+DESCRIPTION_libsvm="${desc_base}libsvm module."
+#DESCRIPTION_pari="${desc_base}pari module."
+DESCRIPTION_pcre="${desc_base}pcre module."
+DESCRIPTION_postgresql="${desc_base}postgresql module."
+DESCRIPTION_zlib="${desc_base}zlib module."
+HOMEPAGE="https://clisp.sourceforge.io/"
+COPYRIGHT="1992-1993 Bruno Haible, Michael Stoll
+	1994-1997 Bruno Haible, Marcus Daniels
+	1998 Bruno Haible, Pierpaolo Bernardi, Sam Steingold
+	1999-2000 Bruno Haible, Sam Steingold
+	2001-2018 Sam Steingold, Bruno Haible
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="6024adde90ea7b8cffb70fd209c282ead6aa19c5"
+SOURCE_URI="https://gitlab.com/gnu-clisp/clisp/-/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="fc865c36416852d27d69b09bc32cc9d1895347e0b1f7d9b7ec4d37de8cb0394f"
+SOURCE_DIR="clisp-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+requires_base="
+	haiku$secondaryArchSuffix
+	clisp$secondaryArchSuffix
+	"
+PROVIDES="
+	clisp$secondaryArchSuffix = $portVersion
+	cmd:clisp = $portVersion
+	cmd:clisp_link = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	libbsd$secondaryArchSuffix
+	libffcall$secondaryArchSuffix
+	libiconv$secondaryArchSuffix
+	libintl$secondaryArchSuffix
+	libncurses$secondaryArchSuffix
+	libnetwork$secondaryArchSuffix
+	libreadline$secondaryArchSuffix >= 8
+	libsigsegv$secondaryArchSuffix
+	libunistring$secondaryArchSuffix
+	"
+PROVIDES_berkeley_db="
+	clisp_berkeley_db$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_berkeley_db="$requires_base
+	lib:db$secondaryArchSuffix
```

Comment:
```suggestion
	lib:libdb$secondaryArchSuffix
```

Reply:
This is the same for the following statements, it's always (well mostly) `lib:libfoo` not `lib:foo`